src/tiled/tileanimationeditor.cpp

```diff
@@ -67,17 +67,20 @@ class FrameListModel : public QAbstractListModel
 
     void setFrames(const Tileset *tileset, const QVector<Frame> &frames);
     void addTileIdAsFrame(int id);
+    void setDefaultFrameTime(int duration);
     const QVector<Frame> &frames() const;
 
 private:
-    static const int DEFAULT_DURATION = 100;
+    static int DEFAULT_DURATION;
```

Comment:
This variable used by the in `ALL_CAPS` because it was a constant. Now that it can be changed, please rename to `mDefaultDuration`.